src/main/java/com/example/n12/fragment/CountFragment.kt

```diff
@@ -0,0 +1,93 @@
+package com.example.n12.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.n12.*
+import com.example.n12.adapter.CountAdapter
+import com.example.n12.databinding.FragmentCountBinding
+import com.example.n12.model.GithubUser
+import com.example.n12.retrofit.GitHubService
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.HttpException
+import retrofit2.Response
+
+class CountFragment : Fragment() {
+    private var _binding: FragmentCountBinding? = null
+    private val binding get() = requireNotNull(_binding)
+    private val adapter by lazy {
+        CountAdapter() {

```

Comment:
здесь на скобках должен быть hint, что они не нужны
```suggestion
        CountAdapter {

```